DbgPack/asset2.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass, field
+from pathlib import Path
+from zlib import decompress
+
+from .abc import AbstractAsset
+from .struct_reader import BinaryStructReader
+
+
+@dataclass
+class Asset2(AbstractAsset):
```

Comment:
Don't forget a docstring:
```suggestion
class Asset2(AbstractAsset):
    """
    Represents an asset entry in a .pack2 file.
    """
```